src/llmq/quorums.cpp

```diff
@@ -502,14 +500,50 @@ std::vector<CQuorumCPtr> CQuorumManager::ScanQuorums(Consensus::LLMQType llmqTyp
         return {};
     }
 
-    const CBlockIndex* pIndexScanCommitments{pindexStart};
+    const CBlockIndex* pindexStore{pindexStart};
+    const auto& llmq_params_opt = GetLLMQParams(llmqType);
+    assert(llmq_params_opt.has_value());
```

Comment:
introduction of assert here is scary no? what if we do 
```suggestion
    if (!llmq_params_opt) return {};
```

Reply:
But do we not think this be able to handle unknown llmqtype and return empty?